src/sentry/api/endpoints/project_release_files.py

```diff
@@ -50,7 +50,12 @@ def get(self, request, project, version):
         )
 
         if query:
-            file_list = file_list.filter(Q(name__icontains=query))
+            names = query.split(",")
```

Comment:
Instead of splitting on `,` could we send the query parameter multiple times so that the endpoint gets a list? It is possible to use `,` in both filenames and URL paths, so we'd need to work around that.